src/index.ts

```diff
@@ -72,6 +72,7 @@ async function run() {
     }
 }
 
+// This also acts as a "kubernetes safe" function, which is why maxLength defaults to 52
```

Comment:
What if we make the actual number in the function arguments a constant?

```suggestion
const KUBERNETES_SAFE_LENGTH = 52
```